app/Http/Controllers/ItemsController.php

```diff
@@ -70,8 +70,18 @@ public function unfaveItem(Request $request) {
     }
 
 
-    public function showUserHome() {
-        return view('welcome');
+    public function showUserHome(Request $request) {
+        $categoryCount = DB::table('items')
+          ->join('categories', 'items.category_id', '=', 'categories.id')
```

Comment:
I'll accept this for now, but would definitely like to see if we can refactor this.